configs/client_cli/config.json

```diff
@@ -9,8 +9,8 @@
     "web_login": "mad_hatter",
     "password": "ilovetea"
   },
-  "TORII_API_URL": "http://127.0.0.1:8080",
-  "TORII_TELEMETRY_URL": "http://127.0.0.1:8180",
+  "TORII_API_URL": "http://127.0.0.1:8080/",
```

Comment:
If this slash is important: 
1. Consider coping with its absence. 
2. Document the necessary slash.